conan/tools/meson/toolchain.py

```diff
@@ -154,7 +155,7 @@ def _add_apple_flags(self):
             return
 
         # SDK path is mandatory for cross-building
-        sdk_path = conanfile.conf["tools.apple:sdk_path"]
+        sdk_path = apple_sdk_path(conanfile)
```

Comment:
I changed this so it takes into account the sdk and sdk_version settings to try to find the path

Reply:
reverted, because you can omit the isysroot flag if not cross-building so this is not necessary